Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-MultipleFailedLogon_FromSameIP.yaml

```diff
@@ -1,6 +1,9 @@
 id: 72727649-6445-46a3-b249-997a009fad89
 name: Failed Logon on SQL Server from Same IPAddress in Short time Span
 description: |
+    This query detects multiple failed logon attempts from the same IP within a short span of time. It relies on the SQLEvent KQL Parser function. 
```

Comment:
This is enough "This query detects multiple failed logon attempts from the same IP within a short span of time. It relies on the SQLEvent KQL Parser function."